--- origsrc/girara-0.1.6/Makefile	2013-05-12 16:09:14.000000000 -0500
+++ src/girara-0.1.6/Makefile	2013-07-30 12:53:48.335521400 -0500
@@ -49,15 +49,15 @@ ${DOBJECTS}: config.mk version.h
 
 ${PROJECT}: static shared
 static: lib${PROJECT}.a
-shared: lib${PROJECT}.so.${SOVERSION}
+shared: cyg${PROJECT}-${SOMAJOR}.dll
 
 lib${PROJECT}.a: ${OBJECTS}
 	$(ECHO) AR rcs $@
 	$(QUIET)ar rcs $@ ${OBJECTS}
 
-lib${PROJECT}.so.${SOVERSION}: ${OBJECTS}
+cyg${PROJECT}-${SOMAJOR}.dll: ${OBJECTS}
 	$(ECHO) LD $@
-	$(QUIET)${CC} -Wl,-soname,lib${PROJECT}.so.${SOMAJOR} -shared ${LDFLAGS} -o $@ ${OBJECTS} ${LIBS}
+	$(QUIET)${CC} -Wl,--out-implib,lib${PROJECT}.dll.a -shared ${LDFLAGS} -o $@ ${OBJECTS} ${LIBS}
 
 clean:
 	$(QUIET)rm -rf ${OBJECTS} ${PROJECT}-${VERSION}.tar.gz \
@@ -128,12 +128,10 @@ install-static: static
 
 install-shared: shared
 	$(ECHO) installing shared library
+	$(QUIET)mkdir -m 755 -p ${DESTDIR}${BINDIR}
 	$(QUIET)mkdir -m 755 -p ${DESTDIR}${LIBDIR}
-	$(QUIET)install -m 644 lib${PROJECT}.so.${SOVERSION} ${DESTDIR}${LIBDIR}
-	$(QUIET)ln -sf lib${PROJECT}.so.${SOVERSION} ${DESTDIR}${LIBDIR}/lib${PROJECT}.so.${SOMAJOR} || \
-		echo "Failed to create lib${PROJECT}.so.${SOMAJOR}. Please check if it exists and points to the correct version of lib${PROJECT}.so."
-	$(QUIET)ln -sf lib${PROJECT}.so.${SOVERSION} ${DESTDIR}${LIBDIR}/lib${PROJECT}.so || \
-		echo "Failed to create lib${PROJECT}.so. Please check if it exists and points to the correct version of lib${PROJECT}.so."
+	$(QUIET)install -m 644 cyg${PROJECT}-${SOMAJOR}.dll ${DESTDIR}${BINDIR}
+	$(QUIET)install -m 644 lib${PROJECT}.dll.a ${DESTDIR}${LIBDIR}
 
 install: options po install-static install-shared install-headers
 		$(QUIET)${MAKE} -C po install
--- origsrc/girara-0.1.6/config.mk	2013-05-12 16:09:14.000000000 -0500
+++ src/girara-0.1.6/config.mk	2013-07-30 12:53:00.053759800 -0500
@@ -17,6 +17,7 @@ SOVERSION = ${SOMAJOR}.${SOMINOR}
 
 # paths
 PREFIX ?= /usr
+BINDIR ?= ${PREFIX}/bin
 LIBDIR ?= ${PREFIX}/lib
 INCLUDEDIR ?= ${PREFIX}/include
 # locale directory
